infrastructure-playbooks/purge-cluster.yml

```diff
@@ -423,18 +423,6 @@
 
   tasks:
 
-  - name: stop ceph.target with systemd
-    service:
-      name: ceph.target
-      state: stopped
-      enabled: no
-    when: ansible_service_mgr == 'systemd'
-
-  - name: check for anything running ceph
```

Comment:
Sure, we can keep this and move it after the package removals. @ktdreyer ensures me that removing these packages should stop all the daemons, and if not it's a packaging bug that needs to be addressed.